ParseLiveQuery/src/main/java/com/parse/ParseLiveQueryClientImpl.java

```diff
@@ -45,6 +51,25 @@
         this.webSocketClientCallback = getWebSocketClientCallback();
     }
 
+    private static URI getDefaultUri() {
+        URL serverUrl = ParseRESTCommand.server;
+        if (serverUrl != null) {
```

Comment:
minor change, but i'd return null serverUrl == null and then take this out of an if block